### ISSUE TITLE
Error while doing cvs2git migration

### ISSUE THREAD

**Issue by anshul14** [mhagger/cvs2svn#5]
Hi mhagger

I am running into an issue with cvs2git migration. Following are the steps that I am following:

I have a Local snapshot of the repository that has CVSROOT and the projects
Then I ran the following command to get the .dat file generated but the files are not generating and I am getting the following error at the Pass 4
Command 
python cvs2git.bat --blobfile=C:\files\git-blob.dat --dumpfile=C:\files\git-dump.dat --username= --encoding=utf8

ERROR: Filtering out excluded symbols and summarizing items... ERROR: The command 'cvs -Q -R -f -d :local:C:\ABC\Work\Migration\FH\Fuzon co -r1.1 -p -kb CVSROOT/checkoutlist' failed with exit status=1 and the following output: cvs checkout: connect to 127.0.0.1(127.0.0.1:2402):2402 failed: No connection could be made because the target machine actively refused it. cvs [checkout aborted]: Couldn't connect to lock server

I tried to connect to the actual cvs server also and then run the command but it still failed. I am using Windows 7 box.

Anyway I can overcome this issue.

**Comment by mhagger** [comment]
Please use the cvs2svn mailing list for such questions.

**Comment by ianier** [comment]
@anshul14: I know I'm (very) late to the party, but did you ever find a solution or workaround for this issue?

**Comment by mhagger** [comment]
I only dimly remember CVS now, but I don't know anything about a CVS lock server. Is that a CVSNT feature? There is no need to lock any files when running cvs2svn, so this doesn't make sense to me. It also needs filesystem-level access to the CVS repository on disk, so it normally should not need to go over the network at all.

**Comment by ianier** [comment]
@mhagger: Sorry, it looks like I posted in wrong issue. The error I got isn't exactly this one but rather [this one](https://github.com/mhagger/cvs2svn/issues/3#issue-46254865). The root cause seems to be, as you correctly mentioned there, that cvs.exe succeeds but writes something to stderr anyway. 
The wrapper workaround does the trick and conversion of my CVSNT repository succeeds, though I haven't imported the result into git yet.
